home/templates/home/index.html

```diff
@@ -441,7 +441,7 @@ <h4 class="feature_sub">Events we conduct </h4>
             <div class="col-md-12 col-sm-12 col-xs-12">
                 <div class="feature_header text-center">
                     <h3 class="feature_title">What <b>We do</b></h3>
-                    <h4 class="feature_sub">Lorem ipsum dolor sit amet, consectetur adipisicing elit. </h4>
+                    <h4 class="feature_sub" align="center"> We are a group of Open Source Enthusiasts who focus on something more preliminary and relevant, ”A Love for Coding”.<br>KOSS plans to hold events such as familiarizing students with terminal commands, Linux environment, Github, teach algorithms and data structures, besides conducting workshops on GUI programming, Web development , Android applications and Hackathons.</h4>
```

Comment:
@madan96 I think he means that 'teaching GitHub` should not make it look that `Git` has been taught or is it trivial in any sense.

Some suggestions.
`terminal commands` -> Unix tools
`algorithms and data structures` -> `Algorithms and Data Structure`
`Android application (development`